recipes/flask-multistatic/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "flask-multistatic" %}
+{% set version = "1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/flask-multistatic-{{ version }}.tar.gz
+  sha256: f5e9c8e451dceb18dc6343a72021c93c5fc509fdf960fa41639c98cd33289b5c
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >3.6
+  run:
+    - flask
+    - python >3.6
+
+test:
+  imports:
+    - flask_multistatic
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pagure.io/flask-multistatic
+  summary: Simple flask plugin to allow overriding static files
+  license: GPL-3.0-or-later
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - mxr-conda
```

Comment:
Could someone from @mxr-conda please confirm they want to be listed as maintainer to this feedstock?

Reply:
@beenje Agree to maintaain